WordPress/Classes/Models/PostType.h

```diff
@@ -0,0 +1,16 @@
+#import <CoreData/CoreData.h>
+
+@class Blog;
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
It seems weird to specify non-null but then annotate each property as `nullable`. Which should it be? Maybe we don't need any of the annotations here?